packages/rln/src/keystore/keystore.spec.ts

```diff
@@ -0,0 +1,311 @@
+        kdf: "pbkdf2",
+        kdfparams: {
+          dklen: 32,
+          c: 1000000,
+          prf: "hmac-sha256",
+          salt: "2dcb5ba5c98fe5e46d961dad36e79a5b"
+        },
+        mac: "2d6e9de6440f52c5db64b13f80399967c8770e82616294e14f40a2e213e7d925"
+      }
+    },
+    "8479C6B9125D43E7B7739F1BAB41779F2F5A4D27FF0E2B6F6CA353032010A22C": {
+      crypto: {
+        cipher: "aes-128-ctr",
+        cipherparams: {
+          iv: "b0eef2c385a04909c4ae9b318e179fa7"
+        },
+        ciphertext:
+          "90b982222072366566fa194be5c170506888e184cadbd52aa38f184ac4e9bc160cc719d809fb6a128e0cbd908e70a71efdc5d51c4dab8aab71e3e6a2ebd9ea4238cb47585137990e896dfa53961bb2b328abfbba82f49db6a9b6e3790cf9e29c145796c6dbf409dc875e7998db827c944a835a29ab4192a11ad1efde5ebdd1a775ecbfefd139c50fbdcbebd6c124d9d65ba6ddaaa83e57695293e7c85dfd6f418d58fa5ffb9ab9b2395c84b57da796d31b6351fde3f1dbab29da6c3f259859bd0719c34f5111a9a12075b53ee91b4598fb2f452dbea823ec094cb757f370b5386a8e5db25cf732681d0cd9bda651ae55cdd125138fd2c8f1ffe87a5eea14df7d355762b37e3e71c33c6fe46a10c2083538910fec12e294de84ff587cab2dd268203699cb180e481f4a3a093b86854cea64341dd9482305abd4a9d7bb304b078bc255bf7cde78689225f17006f24c2cd82d38a59f1e0899965c38fcfd1ec67069143ee05a34922963a527549a002e3221e1461463f573e5f66ba87dcb83a63cb8e3a721c13cd9d4d0c9a0334a558f32027424a5bc9fc12b91981a3f74ac4b62eea3aae8be6c44504696b96afadce5d9222bb67dddf5a7d98dd43d544d79f8720a946c37eba8eb5ae6d70f4bdbbe554cbd4b3abb35ed357c8cb8f55e016ab83bef12bf5c0cdf26c7624c86f16437f545d796addb1aa7370de329930c68b174c871706e7afdf78cc07e0f0c58e45495d0d3bcf3faf9fb6d20369b0adc89766b0c9132677e52112770d017da7658f2a0c0eaeac57416f203700f98bf7b30119407733d4f0bd4322c622120cdf81646c4a1adfb80e757954e41ba0e7816c403b2e4b9ceb2d36e4198921ea719a410ae6f6983e49e7b99c266deb0465af716799e36a5bab70923291da808edeba54267e31e8b64c37123fd45d86e0638",
+        kdf: "pbkdf2",
+        kdfparams: {
+          dklen: 32,
+          c: 1000000,
+          prf: "hmac-sha256",
+          salt: "142a0a65b7f6f480546cc4ef743d7ef9"
+        },
+        mac: "7119b7b78598850de5f6af742e42748a3b005394b6b8b272490f24527ebd8b15"
+      }
+    }
+  }
+};
+
+describe("Keystore", () => {
+  it("shoud create empty store with predefined values", () => {
+    const store = Keystore.create();
+
+    expect(store.toObject()).to.deep.eq({
+      application: "waku-rln-relay",
+      appIdentifier: "01234567890abcdef",
+      version: "0.2",
+      credentials: {}
+    });
+  });
+
+  // TODO: add more thorow credentials testing
```

Comment:
track in an issue?

Reply:
https://github.com/waku-org/js-waku/issues/1850